convert_case.py

```diff
@@ -24,4 +25,23 @@
 if option.lower() == "s":
 	message = message.swapcase()
 
-print("\n" + message)
+if option.lower() == "ou":
+    order = 0
+    counter = 0
+    listO = []
+    for each in message:
```

Comment:
```suggestion
    for char in message:
```

Let's rename the variable `each` to `char`. I think that's more appropriate.

Reply:
What if we just rewrite everything to this? (which matches the alt caps logic, for example, in [https://convertcase.net/](https://convertcase.net/))

```python
if option.lower() == "a":
    message_lower = message.lower()
    counter = 0
    charlist = []

    for char in message_lower:
        counter = -1 if char == " " else counter

        if (counter % 2 != 0):
            charlist.append(char.upper())
        else:
            charlist.append(char)

        counter += 1

    message = ''.join(charlist)
```